bi-connectors/TableauConnector/tableau_support.md

```diff
@@ -26,12 +26,12 @@ Click on **Download** option for downloading `opensearch_sql_odbc.taco` file.
 <full-Tableau-path>\bin\tableau.exe -DDisableVerifyConnectorPluginSignature=true
 ```
 
-* Click on **Connect** > **More** > **SQL by OpenSearch for ES**.
+* Click on **Connect** > **More** > **OpenSearch by OpenSearch Project**.
 
 <img src="img/tableau_select_connector.png" width=600>
 
 * Enter **Server** & **Port** value. 
-* Select required authentication option. For **AWS_SIGV4** authentication, select **Integrated Authentication** and enter value for **Region**. To setup aws access key id and secret key, create ~/.aws/credentials with [AWS CLI](https://aws.amazon.com/cli/) and use `opensearchodbc` as the profile name.
+* Select required authentication option. For **AWS_SIGV4** authentication, select **AWS_SIGv4** and enter value for **Region**. You need to create `~/.aws/credentials` and add `default` profile with aws access key id and secret key.
```

Comment:
question: would ODBC drivers use `default` or still `opensearchodbc` profile? if still `opensearchodbc` i think it's better to keep it documented somewhere

Reply:
Good catch! `ODBC` driver still require `opensearchodbc` profile due to the bug, but Tableau uses `JDBC` driver which works with `default` profile. So there is no mistake.

Reply:
right. after removing that information here, users won't be able to know that `ODBC` driver still require `opensearchodbc` because no other place mentions it

I'm saying should the information be documented somewhere else? maybe in https://github.com/opensearch-project/sql/blob/main/sql-odbc/docs/user/configuration_options.md

Reply:
See 7694813b.